single_cell_analysis/scripts/celltype.vis.R

```diff
@@ -57,15 +57,10 @@ output.dir <- file.path("figures", "features_vis", "simulated_data")
 sim.file.list <- read.table(file.path(input.dir, "sim.file.list.txt"))
 celltye_vis(input.dir, output.dir, sim.file.list, option = "simulated")
 
-
 # visulization for real data
 input.dir <- file.path("features", "real_data")
 output.dir <- file.path("figures", "features_vis", "real_datasets")
 
 # list of real files
 real.file.list <- read.table(file.path(input.dir, "file.list.txt"))
-celltye_vis(input.dir, output.dir, real.file.list, option = "real")
-
-
-
-
+celltye_vis(input.dir, output.dir, real.file.list, option = "real")
```

Comment:
the red arrow means you need to add a new line. There should be 1 empty line at the end of file